ansible/inventories/group_vars/all

```diff
@@ -282,11 +282,8 @@ gpg_home_dir:
 group_config:
 
 grouper:
-  service_name: iplant-grouper.service
-  service_name_short: iplant-grouper
-  compose_service: iplant_grouper
-  service_description: Grouper UI and Web Services
-  image_name: iplant-grouper
+  compose_service: grouper
+  image_repo: "{{ docker.user }}/grouper:2.2.2"
```

Comment:
Most of the settings that were removed here were relics from the systemd deployment that we were using for Grouper. I changed the `image_name` member to `image_repo` to indicate that this is the full name of the repository we'll be using to retrieve the docker image.